test/foundry/OwnableTwoSteps.t.sol

```diff
@@ -48,6 +48,62 @@ contract OwnableTwoStepsTest is TestParameters, TestHelpers, OwnableTwoStepsErro
         ownableTwoSteps.confirmOwnershipTransfer();
         assertEq(ownableTwoSteps.potentialOwner(), address(0));
         assertEq(ownableTwoSteps.owner(), newOwner);
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.NoOngoingTransfer));
+    }
+
+    function testRenounceOwnership() public asPrankedUser(_OWNER) {
+        // 1. Initiate renouncement of ownership
+        vm.expectEmit(false, false, false, true);
+        emit InitiateOwnershipRenouncement();
+        ownableTwoSteps.initiateOwnershipRenouncement();
+        assertEq(ownableTwoSteps.potentialOwner(), address(0));
+        assertEq(ownableTwoSteps.earliestOwnershipRenouncementTime(), block.timestamp + ownableTwoSteps.delay());
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.RenouncementInProgress));
+
+        // 2. Confirm renouncement of ownership
+        // Time travel
+        vm.warp(ownableTwoSteps.earliestOwnershipRenouncementTime());
+        vm.expectEmit(false, false, false, true);
+        emit NewOwner(address(0));
+        ownableTwoSteps.confirmOwnershipRenouncement();
+        assertEq(ownableTwoSteps.potentialOwner(), address(0));
+        assertEq(ownableTwoSteps.owner(), address(0));
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.NoOngoingTransfer));
+    }
+
+    function testCancelTransferOwnership() public asPrankedUser(_OWNER) {
+        address newOwner = address(45);
+
+        // 1. Initiate ownership transfer
+        vm.expectEmit(false, false, false, true);
+        emit InitiateOwnershipTransfer(_OWNER, newOwner);
+        ownableTwoSteps.initiateOwnershipTransfer(newOwner);
+        assertEq(ownableTwoSteps.potentialOwner(), newOwner);
+        assertEq(uint8(ownableTwoSteps.status()), 1);
```

Comment:
Above the enums are converted into uint8, but here it is just a number. Can we use the first style exclusively?

Reply:
Adjusted!